src/qt/EditToolbarHandler.h

```diff
@@ -1,5 +1,5 @@
 #ifndef EDIT_TOOLBAR_HANDLER_H
-#define EDIT_TOOLBAR_HANDELR_H
+#define EDIT_TOOLBAR_HANDLER_H
```

Comment:
TBH I prefer `#pragma once` exactly for this reason

Reply:
Replaced with pragma.